lib/fluent/plugin/in_tail.rb

```diff
@@ -252,6 +254,7 @@ def flush_buffer(tw)
                   else
                     @tag
                   end
+            record[@path_key] ||= tw.path unless @path_key.nil?
```

Comment:
Either is fine for me.
User can check path_key value is not contained or not easily so
additional check is not needed for me.